subsys/net/lib/nrf_cloud/coap/include/dtls.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef DTLS_H_
+#define DTLS_H_
+
+/** @brief Set up DTLS on socket
```

Comment:
```suggestion
/** @brief Set up DTLS on socket.
```
In general, the short function description must end with a full stop.